upstream-community-operators/camel-k/camel-k.v0.3.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,273 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    repository: https://github.com/apache/camel-k
+    containerImage: docker.io/apache/camel-k:0.3.3
+    support: Camel
+    createdAt: 2019-05-06T16:45:00Z
+    certified: "false"
+    description: |-
+      Apache Camel K (a.k.a. Kamel) is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+    alm-examples: |-
+      [{
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "IntegrationPlatform",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "build": {
+            "buildStrategy": "pod"
+          },
+          "resources": {
+            "contexts": [
+              "jvm"
+            ]
+          }
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "Integration",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "source": {
+            "content": "// Add example Java code to create Integration",
+            "name": "Example.java"
+          }
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "IntegrationContext",
+        "metadata": {
+          "name": "example"
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "CamelCatalog",
```

Comment:
Right now CI is creating a `CamelCatalog` example using this configuration which lacks a `spec` section.
Here are the current logs in travis CI: https://travis-ci.com/operator-framework/community-operators/builds/110840458#L415
Are these logs expected when a CamelCatalog is created?

Reply:
`CamelCatalog` CRs do not have a dedicated controller. So it seems the CI logs are expected.

Reply:
I am sorry, but I am not writing the camel operator, you may be commenting on the wrong pull request